test/e2e/specs/updates/updates.go

```diff
@@ -0,0 +1,22 @@
+package updates
+
+import (
+	"io/ioutil"
+
+	"github.com/ghodss/yaml"
+
+	v20180930preview "github.com/openshift/openshift-azure/pkg/api/2018-09-30-preview/api"
+)
+
+// ParseExternalConfig parses an external manifest located at path and returns an external OpenshiftManagedCluster struct
+func ParseExternalConfig(path string) (*v20180930preview.OpenShiftManagedCluster, error) {
```

Comment:
shouldn't be needed in this test

Reply:
shouldn't be needed after I start using the client library to speak to the RP